pyproject.toml

```diff
@@ -148,6 +148,9 @@ ECM_Example = "pybamm.input.parameters.ecm.example_set:get_parameter_values"
 MSMR_Example = "pybamm.input.parameters.lithium_ion.MSMR_example_set:get_parameter_values"
 Chayambuka2022 = "pybamm.input.parameters.sodium_ion.Chayambuka2022:get_parameter_values"
 
+[project.entry-points."pybamm_models"]
+SPM = "pybamm.models.full_battery_models.lithium_ion.spm:SPM"
+
```

Comment:
Should we also add the DFN model, or are you looking to expose just one model for now?